.github/workflows/ci-mac-os.yml

```diff
@@ -68,16 +68,19 @@ jobs:
     
     - name: Test amc2moodle
       run: |
-        python -m amc2moodle.amc2moodle.test
+        python -m amc2moodle.tests.test_amc2moodle
         
     - name: Test moodle2amc
       run: |
         # add automultiplechoice.sty local copy to LaTeX PATH (for this step)
-        export TEXINPUTS=.:./amc2moodle/moodle2amc/test:$TEXINPUTS
-        python -m amc2moodle.moodle2amc.test
+        # export TEXINPUTS=.:./tests/payload_test_amc2moodle/:$TEXINPUTS  (during tests, automultiplechoice.sty is copied in the temporary test folder )
+        python -m amc2moodle.tests.test_moodle2amc
+    - name: Test text parser
+      run: |
+        python -m amc2moodle.tests.test_utils_text
     - name: Test calculated questions parsers
       run: |
-        python -m amc2moodle.utils.test
+        python -m amc2moodle.tests.test_utils_calculatedParser
```

Comment:
It could be a solution but we will lost useful information about the location of the errors on tests (no github actions for hatch exists). We can discuss about this thing in #92.

Reply:
for instance add in `pyproject.toml`
```
[[tool.hatch.envs.test.matrix]]
python = ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```
and run `hatch test -a`

Reply:
I think it is better to use github action for that. You're right.